knowledge-base/hierarchical-breadcrumb-dropdowns-blazor.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Hierarchical Breadcrumb in Blazor
+description: Learn how to customize the Breadcrumb for Blazor by embedding dropdowns within Breadcrumb items to optimize user interaction and functionality.
+type: how-to
+page_title: Hierarchical Breadcrumb in Blazor
+slug: hierarchical-breadcrumb-dropdowns-blazor
+tags: breadcrumb, blazor, customization, dropdown, itemtemplate, hierarchical
+res_type: kb
+ticketid: 1652751
+---
+
+## Environment
+
+| Product |
+| --- |
+| Breadcrumb for Blazor |
```

Comment:
Such a layout is correct, but is inconsistent with the rest of our KBs: https://docs.telerik.com/blazor-ui/knowledge-base/chart-export-to-image